experimental/SymmetricIntersections/Representations.jl

```diff
@@ -0,0 +1,1465 @@
+GG = GAP.Globals
```

Comment:
```suggestion
const GG = GAP.Globals
```
else any code using this is type unstable